### PR TITLE
Support marine-rs-sdk 0.7.0 

### PR DESCRIPTION
**Pull request by ValeryAntopol**

### CODE REVIEW

File: marine/Cargo.toml [hunk 1]
```diff
@@ -11,16 +11,16 @@ name = "marine"
...
 marine-utils = { path = "../crates/utils", version = "0.4.0" }
 marine-rs-sdk-main = { version = "0.6.15", features = ["logger"] }
 marine-rs-sdk = { version = "0.6.15", features = ["logger"] }
 it-json-serde = { path = "../crates/it-json-serde", version = "0.3.1" }
 
-wasmer-runtime = { package = "wasmer-runtime-fl", version = "=0.17.1" }
+wasmer-runtime = { package = "wasmer-runtime-fl", version = "=0.17.1"}
 # dynamicfunc-fat-closures allows using state inside DynamicFunc
-wasmer-core = { package = "wasmer-runtime-core-fl", version = "=0.17.1", features = ["dynamicfunc-fat-closures"] }
+wasmer-core = { package = "wasmer-runtime-core-fl", version = "=0.17.1", features = ["dynamicfunc-fat-closures", "generate-debug-information", "wasm-debug"] }
```
Comment (mikevoronov) [inline marine/Cargo.toml:23]: Why these features are needed?

File: core/src/marine_core.rs [hunk 2]
```diff
@@ -37,6 +39,22 @@ pub struct MarineCore {
...
+impl OutdatedModule {
+    pub fn new(name: String, version: Version, stable_version: Version) -> Self {
+        Self {
+            name,
+            version,
+            stable_version,
+        }
+    }
+}
+
```
Comment (mikevoronov) [inline core/src/marine_core.rs:57]: Suggest to put it to a separate file

File: core/src/marine_core.rs [hunk 3]
```diff
@@ -151,6 +169,26 @@ impl MarineCore {
         records.into()
     }
 
+    pub fn list_outdated(&self) -> Vec<OutdatedModule> {
+        self.modules
+            .iter()
+            .filter_map(|(name, module)| {
+                module
+                    .check_update_status(name.clone())
```
Comment (mikevoronov) [inline core/src/marine_core.rs:177]: I believe that it's better either to rewrite the signature of this function to receive `AsRef<str>` or to pass a reference here.

File: marine/Cargo.toml [hunk 4]
```diff
@@ -40,6 +40,9 @@ thiserror = "1.0.23"
 once_cell = "1.4.0"
 env_logger = "0.7.1"
 pretty_assertions = "0.7.2"
+fluence-keypair = "0.7.0"
+libp2p-core = { version = "0.32.1", default-features = false, features = [ "secp256k1" ] }
+rusqlite = "0.26.1"
```
Comment (mikevoronov) [inline marine/Cargo.toml:45]: Do we need all of this?

File: marine/Cargo.toml (hunk 1 above)
Comment (ValeryAntopol) [reply marine/Cargo.toml:23]: I tried using them to analyze memory usage before i found memory leak. Now they are not needed.

File: core/src/marine_core.rs (hunk 2 above)
Comment (ValeryAntopol) [reply core/src/marine_core.rs:57]: We decided to completely remove it, so I'll do it

File: core/src/marine_core.rs (hunk 3 above)
Comment (ValeryAntopol) [reply core/src/marine_core.rs:177]: ditto

File: marine/Cargo.toml (hunk 4 above)
Comment (ValeryAntopol) [reply marine/Cargo.toml:45]: No, these one were used in debug

File: marine-js/src/module/marine_module.rs [hunk 5]
```diff
@@ -94,6 +94,10 @@ impl MModule {
         let wasmer_instance = WasmerInstance::new(&mit, Rc::new(name.to_string()));
         let it_instance = Arc::new(ITInstance::new(&wasmer_instance, &mit)?);
         let (export_funcs, export_record_types) = Self::instantiate_exports(&it_instance, &mit)?;
+        if let Ok(initialize_func) = wasmer_instance.exports.get("_initialize") {
```
Comment (folex) [inline marine-js/src/module/marine_module.rs:97]: Maybe use `INITIALIZE_FUNC` constant instead of `_initialize`?

**Review by folex — APPROVED** [review]

**Pull request merged**